cmd/cosign/cli/options/registry.go

```diff
@@ -85,6 +86,7 @@ func (o *RegistryOptions) GetRegistryClientOpts(ctx context.Context) []remote.Op
 			google.Keychain,
 			authn.NewKeychainFromHelper(ecr.NewECRHelper(ecr.WithLogger(ioutil.Discard))),
 			authn.NewKeychainFromHelper(credhelper.NewACRCredentialsHelper()),
+			authn.NewKeychainFromHelper(alibabaacr.NewACRHelper().WithLoggerOut(ioutil.Discard)),
```

Comment:
It's not really your responsibility, but I'd like to document this behavior a little better in general.

Along with that, the `--k8s-keychain` flag that controls this is somewhat poorly named -- it always has been, but it's especially confusing if we add a cred helper that isn't included in the standard k8s cred helpers, and it'll only get worse if we add more in the future.

Again, this isn't really your fault, and I don't think we need to change anything in this PR, just noting it for whomever decides to take this on in the future (likely me).